apis/python/tests/test_registration_mappings.py

```diff
@@ -1094,3 +1094,7 @@ def test_registration_with_batched_reads(tmp_path, soma_larger, use_small_buffer
     )
 
     assert len(rd.obs_axis.data) == 1000
+
+
+def test_ealm_expose():
+    """Checks that this is exported from tiledbsoma.io._registration"""
```

Comment:
Is there something missing here?

Reply:
it does the import and if the import fails it'll throw

Reply:
.... oh crap, the pre-commit hook removed that line :headdesk:

Reply:
I believe `import whatever as _` (or in this case, `from what import ever as _`) will also preserve a side-effect import. Alternately, since `tiledbsoma.io` is already imported by this file, you could just say `assert tiledbsoma.io.ExperimentLabelMapping is not None`.